### ISSUE TITLE
apidocs build is occasionally failing

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8501]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8501|
|Type|defect|
|Created|2016-06-24 22:28:41Z|
|Branch|https://github.com/twisted/twisted/tree/8501-rodrigc-apidocs|

Occasionally the apidocs build is failing with new errors.
For example:
https://buildbot.twistedmatrix.com/builders/documentation/builds/445/steps/api-documentation/logs/new%20pydoctor%20errors

```
twisted.internet.pollreactor.PollReactor._poller: invalid ref to poll
twisted.internet.pollreactor.PollReactor: invalid ref to poll
```

<details><summary>Searchable metadata</summary>

```
trac-id__8501 8501
type__defect defect
reporter__rodrigc rodrigc
priority__normal normal
milestone__None None
branch__8501_rodrigc_apidocs 8501-rodrigc-apidocs
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1466807321287193 1466807321287193
changetime__1466976494476438 1466976494476438
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/216

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban set owner to @rodrigc|
|-|-|

Looks great. Many thanks.

I would like to see zero error in the documentation build, so that we can run it on local branch. This is a great step forward.

Please see the minor comment about L{original} vs L{originalAttribute} inside the PR.

Please check my comment and merge.

Thanks!

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub set status to `closed`|
|-|-|

In changeset d85cf083fc6083f9001f345554aae1008a787a8f
```
#!CommitTicketReference repository="" revision="d85cf083fc6083f9001f345554aae1008a787a8f"
Merge 8501-rodrigc-apidocs: Fix references in a few docstrings

Author: rodrigc
Reviewer: adiroiban, alex
Fixes: #8501
```
